kod/object/active/holder/nomoveon/battler/player.kod

```diff
@@ -9902,7 +9902,11 @@ messages:
       return;
    }
 
-   BreakTrance(event = 0, what = $)
+   BreakTrance(event = 0, castwhat = $, usewhat = $)
+   "Sends a BreakTrance message to any trance-holding enchantments."
+   "The event is the action causing the broken trance."
+   "If the event is the player casting a spell then castwhat is the ID for that spell."
```

Comment:
I'm not sure this comment is correct.  It says that castwhat is the ID for the spell, but I think it's actually the spell object itself, not the SID_ code.  Similarly for usewhat: it's the item, not the ID for an item.